Makefile

```diff
@@ -172,6 +172,37 @@ ifeq ($(BUILD_ENTERPRISE_READY),true)
 	docker-compose -f docker-compose.yaml up haproxy
 endif
 
+run-multi: run-client
+ifeq ($(BUILD_ENTERPRISE_READY),true)
+	@echo Removing possibly wrong configured default network
+	docker network rm mattermost-server_mm-test
+
+	@echo Adding necessary host alias in /etc/hosts
+
+	./scripts/update_hosts
+
+	@echo Running two multi mattermost instances
+	@echo the multi UIs will be available in
+	@echo - http://org-a.local:8065
+	@echo - http://org-b.local:8065
+	docker-compose -f docker-compose.multi.yaml up shared_proxy
```

Comment:
Should we honour `RUN_SERVER_IN_BACKGROUND` here and optionally `--detach` if `true`? (I prefer running in the foreground myself, but just wondering about uniformity with the rest of the tooling.)

Reply:
No strong pref here either, we currently defaulting to `false` here: https://github.com/mattermost/mattermost-server/pull/16867/files#diff-33d1f28445574997ded75f23c6499642cdc41aba2296da8d31b5ee0645d0f2c8R111

Have in mind that the server is running within docker in the HA and Multi setup. Whether in `make run` it runs on the host.